docs/user-guide/how_to_use_volcano_vgpu.md

```diff
@@ -32,106 +70,122 @@ data:
+          deviceshare.SchedulePolicy: binpack  # scheduling policy. binpack / spread
 ```
 
-### Install Volcano device plugin
-
-Please refer to [volcano vgpu device plugin](https://github.com/Project-HAMi/volcano-vgpu-device-plugin?tab=readme-ov-file#enabling-gpu-support-in-kubernetes)
-
-### Verify environment is ready
-
-Check the node status, it is ok if `volcano.sh/vgpu-memory` and `volcano.sh/vgpu-number` are included in the allocatable resources.
-
-```shell script
-$ kubectl get node {node name} -oyaml
-...
-status:
-  addresses:
-  - address: 172.17.0.3
-    type: InternalIP
-  - address: volcano-control-plane
-    type: Hostname
-  allocatable:
-    cpu: "4"
-    ephemeral-storage: 123722704Ki
-    hugepages-1Gi: "0"
-    hugepages-2Mi: "0"
-    memory: 8174332Ki
-    pods: "110"
-    volcano.sh/vgpu-memory: "89424"
-    volcano.sh/vgpu-number: "8"    # GPU resource
-  capacity:
-    cpu: "4"
-    ephemeral-storage: 123722704Ki
-    hugepages-1Gi: "0"
-    hugepages-2Mi: "0"
-    memory: 8174332Ki
-    pods: "110"
-    volcano.sh/vgpu-memory: "89424"
-    volcano.sh/vgpu-number: "8"   # GPU resource
+Check with:
+
+```bash
+kubectl get node {node-name} -o yaml
 ```
 
-### Running GPU Sharing Jobs
+---
 
-VGPU can be requested by both set "volcano.sh/vgpu-number" , "volcano.sh/vgpu-cores" and "volcano.sh/vgpu-memory" in resource.limit
+### HAMI-core Usage
 
-```shell script
-$ cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: Pod
+* **Pod Spec**:
+
+```yaml
 metadata:
-  name: gpu-pod1
+  name: hami-pod
+  annotations:
+    volcano.sh/vgpu-mode: "hami-core"
 spec:
   schedulerName: volcano
   containers:
-    - name: cuda-container
-      image: nvidia/cuda:9.0-devel
-      command: ["sleep"]
-      args: ["100000"]
-      resources:
-        limits:
-          volcano.sh/vgpu-number: 2 # requesting 2 gpu cards
-          volcano.sh/vgpu-memory: 3000 # (optinal)each vGPU uses 3G device memory
-          volcano.sh/vgpu-cores: 50 # (optional)each vGPU uses 50% core  
-EOF
+  - name: cuda-container
+    image: nvidia/cuda:9.0-devel
+    resources:
+      limits:
+        volcano.sh/vgpu-number: 1    # requesting 1 gpu cards
+        volcano.sh/vgpu-cores: 50    # (optional)each vGPU uses 50%
+        volcano.sh/vgpu-memory: 3000 # (optional)each vGPU uses 3G GPU memory
 ```
 
-You can validate device memory using nvidia-smi inside container:
+---
+
+### Dynamic MIG Usage
 
-![img](https://github.com/Project-HAMi/volcano-vgpu-device-plugin/blob/main/doc/hard_limit.jpg)
+* **Enable MIG Mode**:
 
-> **WARNING:** *if you don't request GPUs when using the device plugin with NVIDIA images all
-> the GPUs on the machine will be exposed inside your container.
-> The number of vgpu used by a container can not exceed the number of gpus on that node.*
+```bash
+sudo nvidia-smi -mig 1
+```
 
-### Monitor
+* **Geometry Config**:
+  Define MIG slice geometries in a `volcano-vgpu-device-config` ConfigMap in `kube-system` namespace.
```

Comment:
We can add more details here. By default, users can modify the MIG configuration by modifying the `volcano-vgpu-device-config` configmap under `kube-system` ns, but users can also modify the `deviceshare.KnowGeometriesCMName` and `deviceshare.KnowGeometriesCMNamespace` parameters to modify the name of the configmap and the ns where it is located.

Reply:
remove this configuration for the hami device plugin doesnot support configuration. Leave it to the future once the dp change.